### ISSUE TITLE
PHP for nginx

### ISSUE THREAD

**Issue by brrr2** [tokyoshells/issues#40]
Can you enable PHP (and the SQLite module for it) for nginx?

**Comment by an-empty-string** [comment]
I'll work on that later today.

**Comment by brrr2** [comment]
I think setting up PHP is significantly easier with Apache. I've never used nginx before.

**Comment by an-empty-string** [comment]
It is, I've almost got it working with nginx.
Also I have a very strange definition of "later today" and should probably be better about that, oops.

**Comment by sckasturi** [comment]
@fwilson42 can this be closed?

**Comment by an-empty-string** [comment]
not really. I no longer have the time to do serious work on tokyoshells unfortunately, though.

**Comment by vijaiaeroastro** [comment]
php-fpm works well with nginx doesn't it?
